test_vew_model.py

```diff
@@ -0,0 +1,140 @@
+todo_item_2_name = "todo item 2"
+todo_item_2_id = "a4f3f21d-e04d-42e6-a6ed-63290b80d654"
+todo_item_2_desc = "description for second todo item"
+todo_item_2_due = "2020-10-24T02:02:02.000000Z"
+todo_item_2_last_activity = "2020-10-20T02:02:02.000000Z"
+doing_item_1_name = "doing item 1"
+doing_item_1_id = "2f464251-68a7-4307-973e-c17cb1acc0a3"
+doing_item_1_desc = "description for first doing item"
+doing_item_1_due = "2020-10-24T03:03:03.000000Z"
+doing_item_1_last_activity = "2020-10-20T03:03:03.000000Z"
+doing_item_2_name = "doing item 2"
+doing_item_2_id = "fa506c9a-a8a0-4ef5-9790-789350c9d551"
+doing_item_2_desc = "description for second doing item"
+doing_item_2_due = "2020-10-24T04:04:04.000000Z"
+doing_item_2_last_activity = "2020-10-20T04:04:04.000000Z"
+done_item_1_name = "done item 1"
+done_item_1_id = "f5ffb57d-d2cc-481d-bfaa-46c5349de175"
+done_item_1_desc = "description for first done item"
+done_item_1_due = "2020-10-24T05:05:05.000000Z"
+done_item_1_last_activity = "2020-10-20T01:01:01.000000Z"
+done_item_2_name = "done item 2"
+done_item_2_id = "16c4cae4-e7ef-4a32-8677-fefc0a8406fa"
+done_item_2_desc = "description for second done item"
+done_item_2_due = "2020-10-24T06:06:06.000000Z"
+done_item_2_last_activity = "2020-10-19T23:59:59.000000Z"
+
+todo_item_1 = TodoItem({"name": todo_item_1_name, "id": todo_item_1_id, "desc": todo_item_1_desc, "due": todo_item_1_due, "dateLastActivity": todo_item_1_last_activity}, Status.TODO)
+todo_item_2 = TodoItem({"name": todo_item_2_name, "id": todo_item_2_id, "desc": todo_item_2_desc, "due": todo_item_2_due, "dateLastActivity": todo_item_2_last_activity}, Status.TODO)
+doing_item_1 = TodoItem({"name": doing_item_1_name, "id": doing_item_1_id, "desc": doing_item_1_desc, "due": doing_item_1_due, "dateLastActivity": doing_item_1_last_activity}, Status.DOING)
+doing_item_2 = TodoItem({"name": doing_item_2_name, "id": doing_item_2_id, "desc": doing_item_2_desc, "due": doing_item_2_due, "dateLastActivity": doing_item_2_last_activity}, Status.DOING)
+done_item_1 = TodoItem({"name": done_item_1_name, "id": done_item_1_id, "desc": done_item_1_desc, "due": done_item_1_due, "dateLastActivity": done_item_1_last_activity}, Status.DONE)
+done_item_2 = TodoItem({"name": done_item_2_name, "id": done_item_2_id, "desc": done_item_2_desc, "due": done_item_2_due, "dateLastActivity": done_item_2_last_activity}, Status.DONE)
+
+
+@pytest.fixture
+def view_model():
+    items = [todo_item_1, todo_item_2, doing_item_1, doing_item_2, done_item_1, done_item_2]
+    view_model = ViewModel(items)
+    return view_model
+
+
+def test_get_todo_items(view_model):
+    
+    # Act
+    todo_items = view_model.todo_items
+
+    # Assert
+    assert len(todo_items) == 2
+    assert todo_item_1 in todo_items
+    assert todo_item_2 in todo_items
+    assert doing_item_1 not in todo_items
+    assert doing_item_2 not in todo_items
+    assert done_item_1 not in todo_items
+    assert done_item_2 not in todo_items
+
```

Comment:
What's the point of all of these `not` assertions when you already know the length of the list is two and exactly what the two elements are?

Same for the other tests.